input/fsh/profiles.fsh

```diff
@@ -0,0 +1,108 @@
+Profile: AuthorizationIdentifier
```

Comment:
I would in general prefer to have each profile in separate files (comment also apply to extensions, invariants, ...)

Reply:
ok - so ```file-per-definition``` https://fshschool.org/docs/gofsh/running/#style  ?

Reply:
Dont't know GoFSH that well

Reply:
Its now split across multiple files ... I can't say that I favour that layout ...